xml/deploy_cephadm.xml

```diff
@@ -1074,23 +1099,66 @@ config: OK
 <screen>&prompt.smaster;ceph-salt apply --non-interactive</screen>
    </tip>
   </sect2>
-
-  <sect2 xml:id="day2-deployment">
-   <title>Deploying Services and Gateways</title>
-   <para>
-    After you have deployed the basic &ceph; cluster, you need to deploy core
-    services to more cluster nodes. To make the cluster data accessible to
-    clients, deploy additional services as well.
-   </para>
-   <para>
-    Currently, we support deployment of &ceph; services on the command line by
-    using the &ceph; orchestrator (<command>ceph orch</command> subcommands).
-   </para>
+  <sect2 xml:id="deploy-min-cluster-final-steps">
+    <title>Final Steps</title>
+    <para>
+      Once the <command>ceph-salt apply</command> command has completed,
+      you should have one &mon; and one &mgr; running on the
+      cluster's bootstrap minion. You should be able to run
+      the <command>ceph status</command> command successfully on any of the
+      minions that were given the <literal>admin</literal> role, either as <literal>root</literal>
+      or the <literal>cephadm</literal> user.
+    </para>
+    <para>
+      The next steps involve using the &cephadm; to deploy additional &mon;,
+      &mgr;, OSDs, the Monitoring Stack, and Gateways. These steps are
+      collectively referred to as the "post-bootstrapping phase", or sometimes
+      "Day 2 deployment".
```

Comment:
Day 2 is operations.  At least according to all the sources that I can find.  Day 1 is deployment.  I don't know that having only a single monitor and manager is considered a successful deployment from a customer perspective.

Reply:
The `ceph orch apply` commands for initially deploying services and daemons are identical to the commands used in day-to-day operations. I guess that's why the orchestrator folks refer to this deployment phase as "Day 2".

Be that as it may, I'm fine to drop all mention of days and numbers thereof, in order to avoid conclusion. "Bootstrap" and "post-bootstrap" deployment phases are plenty fine for our terminological needs, and hopefully impossible to misconstrue.

Reply:
Going to remove it completely so it simply says "The next steps involve using the &cephadm; to deploy additional &mon;,  &mgr;, OSDs, the Monitoring Stack, and Gateways." that's all the customer needs to know